client/views/tickets/ticket_page.html

```diff
@@ -24,6 +24,10 @@
     </div>
     {{#if noTesting}}
       <div class="no-testing-ticket">NO TESTING REQUIRED</div>
+      {{#if userAdmin}}
```

Comment:
Instead of duplicating this code, we should remove the else statement below, and move the `{{#if userAdmin}}` statement out of the  `{{#if noTesting}}` block:
```html
</div>
    {{#if noTesting}}
      <div class="no-testing-ticket">NO TESTING REQUIRED</div>
    {{/if}}
    {{#if userAdmin}}
      <a href="{{pathFor 'ticketEdit'}}" class="btn-edit">edit</a>
      <div class="delete-ticket ticket-page">Delete</div>
    {{/if}}
...
```